### ISSUE TITLE
When open xlsx with thumbnail it has some warning.

### ISSUE THREAD

**Issue by mix5003** [PHPOffice/PhpSpreadsheet#2516]
This is:

```
- [x] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
It can open xlsx with thumbnails files without any warning. (like phpoffice/phpspreadsheet 1.16.0)

### What is the current behavior?
it warning. stacktrace look likes
```
[2022-01-21 12:39:57] local.ERROR: simplexml_load_string(): Entity: line 1: parser error : Start tag expected, '&lt;' not found {"userId":6,"exception":"[object] (ErrorException(code: 0): simplexml_load_string(): Entity: line 1: parser error : Start tag expected, '&lt;' not found at /path/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Reader/Xlsx.php:126)
[stacktrace]
#0 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleError()
#1 /path/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Reader/Xlsx.php(126): simplexml_load_string()
#2 /path/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Reader/Xlsx.php(232): PhpOffice\\PhpSpreadsheet\\Reader\\Xlsx->loadZip()
...
```

### What are the steps to reproduce?
[file_with_thumbails.xlsx](https://ishare.in.th/tmp/PhpSpreadsheet-2516/file_with_thumbails.xlsx)
```php
<?php

require __DIR__ . '/vendor/autoload.php';

use PhpOffice\PhpSpreadsheet\IOFactory;

$reader = IOFactory::createReader('Xlsx');
$reader->listWorksheetInfo('file_with_thumbails.xlsx');
```

### Which versions of PhpSpreadsheet and PHP are affected?
PHP 7.4.9
phpoffice/phpspreadsheet 1.21.0

**Comment by Decoydoll** [comment]
I got the same error for an `.xlxs` file. But it was an error instead of a warning, so an exception was thrown. I tried to save the file as an `.xls` with MS Excel and then it worked fine. Here is the [file](https://docs.google.com/spreadsheets/d/1MPrZABCox1Pz3d_CD1Cr7Qe-me4nFuZl/edit?usp=sharing&ouid=111296775688870484870&rtpof=true&sd=true) 

Which versions of PhpSpreadsheet and PHP are affected?
PHP 7.4.27
phpoffice/phpspreadsheet 1.21.0
